unitylibs/core/workflow/workflow-acrobat/action-binder.js

```diff
@@ -176,39 +176,36 @@ export default class ActionBinder {
   };
    
 static ERROR_MAP = {
-  'verb_upload_error_generic': -1,
-  'verb_upload_error_loading_verb_limits': -50,
-  'verb_upload_error_empty_verb_limits': -51,
-  'verb_upload_error_duplicate_asset': -52,
-  'verb_upload_error_validate_files': -100,
-  'verb_upload_error_renaming_file' : -101,
-  'verb_upload_error_max_page_count': -150,
-  'verb_upload_error_min_page_count': -151,
-  'verb_upload_error_verify_page_count': -152,
-  'verb_upload_error_max_page_count_multi': -153,
-  'verb_upload_error_unsupported_type': -170,
-  'verb_upload_error_empty_file': -171,
-  'verb_upload_error_file_too_large': -172,
-  'verb_upload_error_only_accept_one_file': -173,
-  'verb_upload_error_file_same_type': -174,
-  'verb_upload_error_unsupported_type_multi': -200,
-  'verb_upload_error_empty_file_multi': -201,
-  'verb_upload_error_file_too_large_multi': -202,
-  'verb_upload_error_multiple_invalid_files': -203,
-  'verb_upload_error_max_num_files': -204,
-  'verb_upload_error_max_quota_exceeded': -250,
-  'verb_upload_error_no_storage_provision': -251,
-  'verb_upload_error_duplicate_operation': -252,
-  'verb_upload_exception_finalize': -300,
-  'verb_upload_exception_validate_page_count': -301,
-  'verb_upload_error_fetch_redirect_url': -350,
-  'verb_upload_error_finalize': -351,
-  'verb_upload_error_chunk_upload': -352,
-  'verb_cookie_not_set': -353,
-  'verb_upload_warn_chunk_upload': -600,
-  'verb_upload_warn_delete_asset': -601,
-  'verb_upload_error_redirect_to_app': -900,
-  'verb_upload_error_finalize_asset': -901
+  'error_generic': -1,
+  'pre_upload_error_loading_verb_limits': -50,
+  'pre_upload_error_empty_verb_limits': -51,
+  'pre_upload_error_renaming_file' : -52,
+  'pre_upload_error_fetch_redirect_url': -53,
+  'validation_error_validate_files': -100,
+  'validation_error_unsupported_type': -101,
+  'validation_error_empty_file': -102,
+  'validation_error_file_too_large': -103,
+  'validation_error_only_accept_one_file': -104,
+  'validation_error_file_same_type': -105,
+  'validation_error_unsupported_type_multi': -200,
+  'validation_error_empty_file_multi': -201,
+  'validation_error_file_too_large_multi': -202,
+  'validation_error_multiple_invalid_files': -203,
+  'validation_error_max_num_files': -204,
+  'upload_validation_error_max_page_count': -300,
+  'upload_validation_error_min_page_count': -301,
+  'upload_validation_error_verify_page_count': -302,
+  'upload_validation_error_max_page_count_multi': -303,
+  'upload_validation_error_duplicate_asset': -304,
+  'upload_error_max_quota_exceeded': -400,
+  'upload_error_no_storage_provision': -401,
+  'upload_error_chunk_upload': -402,
+  'upload_error_finalize_asset': -403,
+  'upload_error_redirect_to_app': -500,
+  'warn_cookie_not_set': -600,
```

Comment:
@rohitsahu As discussed, let's go ahead and remove this `warn_cookie_not_set` and its accompanying logic in the action-binder. This is no longer relevant and is an unnecessary check.

@Ruchika4 Do you agree? Please let us know if any concerns.

Reply:
Thanks for confirming @Ruchika4!